src/util/useLibraryOptions.ts

```diff
@@ -33,6 +39,6 @@ export default function useLibraryOptions(): IUseLibraryOptions {
     // eslint-disable-next-line eqeqeq
     const active = !(unread == undefined);
```

Comment:
add the active condition here as well.

Reply:
I will do that, but i am closing this pull request. I screwed up with git so i am closing this and making another pull request. I am so sorry